### PR TITLE
EVE: Patch Github Call

### PR DESCRIPTION
**Pull request by haydenudelson**
#### What does this PR do?

Changes Github authentication to increase rate limit.  Rate limit should now be 5000 calls/hr up from 60 calls/hour.  Hopefully this will be enough for the Wallboard.

#### Who is reviewing it?

@vinamartin @alchucam @mcalcote @haydenudelson

@gordocanchola

#### How should this be tested?

`yarn go` to test locally or `make go` to test in Docker.  You'll need the GITHUB_TOKEN environment variable, which I can send to you over Slack.

#### Checklist:
- [✅] Documentation Updated

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist.
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -28,12 +28,12 @@ pipeline {
           string(credentialsId: 'SLACK_CHANNEL', variable: 'SLACK_CHANNEL'),
           string(credentialsId: 'GITHUB_CLIENT_ID', variable: 'GITHUB_CLIENT_ID'),
           string(credentialsId: 'GITHUB_CLIENT_SECRET', variable: 'GITHUB_CLIENT_SECRET'),
```
Comment (alchucam) [inline Jenkinsfile]: ✏️ you can removed the two lines of importing GITHUB_CLIENT_ID and GITHUB_CLIENT_SECRET as we no longer need them.

**Comment by alchucam** [comment]
❓ 
image
It shows this error during local run. Interestingly, it works fine in dockerized mode. Any Idea what causes the difference?

File: client/components/Github.js [hunk 2]
```diff
@@ -68,7 +66,6 @@ const PRSubline = styled.div`
 
 const Description = styled.p`
   margin: 0 0 0 0;
-  max-height: 48px;
```
Comment (alchucam) [inline client/components/Github.js]: ❓  Why is the max-height been removed?

Comment (haydenudelson) [reply client/components/Github.js]: It's redundant, since there is a line below it that limits the number of lines to 2.

**Pull request merged**